services/movie.py

```diff
@@ -1,14 +1,19 @@
+from django.db import transaction
 from django.db.models import QuerySet
 
 from db.models import Movie
 
 
 def get_movies(
+    title: str = None,
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
```

Comment:
The type annotations for `genres_ids` and `actors_ids` should be `Optional[list[int]]` to indicate that these parameters can be `None`.